common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java

```diff
@@ -70,6 +77,16 @@ public abstract class GrpcClient extends RpcClient {
     
     private static final long DEFAULT_KEEP_ALIVE_TIME = 6 * 60 * 1000;
     
+    private Properties configProperties = new Properties();
+    
+    private final Long defaultTimeOut = 3000L;
+    
+    private final int queueSize = 10000;
+    
+    private final Long keepAlive = 10L;
```

Comment:
keepAlive and defaultTimeOut is only used by `String.valueOf` should we define them as String directly?

And the constants I think should as `static final`

one more about the `queueSize` should we make it can config also?

Reply:
@KomachiSion 按你建议修改了一下代码，加载配置顺序为JVM 参数 --> 构造器传入Properties ---> Default 配置